src/main/java/ch/uzh/soprafs22/groupmatcher/model/Question.java

```diff
@@ -30,7 +30,7 @@ public class Question {
 
     @Column(nullable = false)
     @Enumerated(EnumType.STRING)
-    private QuestionCategory questionCategory = QuestionCategory.KNOWLEDGE;
+    private QuestionCategory questionCategory = QuestionCategory.SIMILARITYMATCHING;
```

Comment:
You can remove the default value here as well

Add this function at this class (after the list of answers):

```
   public Double calculateMatchingScore() {
        return switch (matcher.getMatchingStrategy()) {
            case MOST_SIMILAR -> 1.0;
            case BALANCED_SKILLS -> questionCategory.getBalancingFactor();
        };
    }
```

Also make sure to:
1. Remove LEAST_SIMILAR from the MatchingStrategy enum, so that you will only have MOST_SIMILAR, BALANCED_SKILLS
2. Add this at the end of the Answer entity module:

```
    public Double calculateMatchingScore() {
        return question.calculateMatchingScore();
    }
```